docs/docs.trychroma.com/markdoc/content/production/deployment.md

```diff
@@ -1,45 +0,0 @@
-# Deployment
```

Comment:
What happens when we delete files like this? We should be setting up a 301 redirect to a new route carefully. (Can we specify those redirects in a vercel config file?)